src/templates/combat/combatant.hbs

```diff
@@ -0,0 +1,50 @@
+<li class="combatant actor directory-item flexrow {{this.css}}" data-combatant-id="{{this.id}}">
+    <img class="token-image" data-src="{{this.img}}" alt="{{this.name}}" />
+    <div class="token-name flexcol">
+        <h4>{{this.name}}</h4>
+        <div class="combatant-controls flexrow">
+            {{#if ../user.isGM}}
+            <a class="combatant-control {{#if this.hidden}}active{{/if}}" aria-label="{{localize 'COMBAT.ToggleVis'}}"
+                role="button" data-tooltip="COMBAT.ToggleVis" data-control="toggleHidden">
+                <i class="fas fa-eye-slash"></i>
+            </a>
+            <a class="combatant-control {{#if this.defeated}}active{{/if}}"
+                aria-label="{{localize 'COMBAT.ToggleDead'}}" role="button" data-tooltip="COMBAT.ToggleDead"
+                data-control="toggleDefeated">
+                <i class="fas fa-skull"></i>
+            </a>
+            {{/if}}
```

Comment:
This doesn't appear to work for me. I'm connected as a GM user but I don't see the visibility toggle. Maybe you need to `@root.user.isGM`? Or perhaps the context isn't getting pass through properly because it's a partial.

Reply:
This appears to have been something between them being in an each block and a partial. Adding @root to the appropriate sections resolved the issue.